docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,189 @@
+# Deploying Subnets on Production Infrastructure
+
+## Introduction
+
+After architecting your Subnet environment on the [local machine](create-a-local-subnet.md), proving the design and testing it out on [the testnet](create-a-fuji-subnet.md), eventually you will need to deploy your Subnet to production environment. Running a subnet in production is much more involved than local and testnet deploys, as your Subnet will have to take care of real world usage, maintaining uptime, upgrades and all of that in a potentially adversarial environment. The purpose of this document is to point out a set of general considerations and propose potential solutions to them.
+
+The architecture of the environment your particular Subnet will use will be greatly influenced by the type of load and activity your Subnet is designed to support so your solution will most likely differ from what we propose here. Still, it might be useful to follow along, to build up the intuition for the type of questions you will need to consider.
+
+## Node setup
+
+Nodes are essential elements for running your Subnet in production. At a minimum, your Subnet will need validator nodes, potentially also nodes that act as RPC servers, indexers or explorers. Running a node is basically running an instance of [AvalancheGo](https://github.com/ava-labs/avalanchego) on a server.
+
+### Server OS
+
+Although AvalancheGo can run on a MacOS or a Windows computer, we strongly recommend running nodes on computers running Linux as they are designed specifically for server loads and all the tools and utilities needed for administering a server are native to Linux.
+
+### Hardware specification
+
+For running AvalancheGo as a validator on the Primary Network the recommended configuration is as follows:
+
+* CPU: Equivalent of 8 AWS vCPU
+* RAM: 16 GiB
+* Storage: 1TB with at least 3000 IOPS
+* OS: Ubuntu 18.04/20.04
+* Network: Reliable IPv4 or IPv6 network connection, with an open public port
+
+That is the configuration sufficient for running a Primary Network node. Any resource requirements for your Subnet come on top of this, so you should not go below this configuration, but may need to step up the specification if you expect your Subnet to handle a significant amount of transactions.
+
+Be sure to set up monitoring of resource consumption for your nodes because resource exhaustion may cause your Subnet to slow down or even halt.
```

Comment:
```suggestion
Be sure to set up monitoring of resource consumption for your nodes because resource exhaustion may cause your node to slow down or even halt, which may severely impact your Subnet negatively.
```

Reply:
Yep, good call.